client/components/resource-details/ResourceInfo.tsx

```diff
@@ -144,29 +137,164 @@ const Reviews = ({ reviews }: ReviewsProps) => {
+  margin-left: auto;
+  align-self: flex-start;
+  @media ${smallerThan(width.mobileS)} {
+    margin-left: 24px;
+  }
+`
+
+const ReviewTopRow = styled.div`
+  display: flex;
+  @media ${smallerThan(width.laptop)} {
+    flex-wrap: wrap;
+  }
+`
+
+const ShowOnLaptop = styled.div`
+  @media ${smallerThan(width.laptop)} {
+    display: none;
+  }
 `
 
-const Rating = styled.p``
+const HideOnLaptop = styled.div`
+  @media ${largerThan(width.laptop)} {
+    display: none;
+  }
+`
+
+const AuthorInfo = styled.div`
+  font-style: italic;
+`
 
-const ReviewEntry = ({ review }: { review: Review }) => {
+const Comment = styled(({ className, review }: { className?: string; review: Review }) => {
   return (
-    <>
-      <Avatar />
-      <Description />
-      <Rating />
-    </>
+    <div className={className}>
+      <Description>{review.comment}</Description>
+      <AuthorInfo>
+        — {Faculty[review.faculty]} Student, {timeSince(review.timestamp)} ago
+      </AuthorInfo>
+    </div>
   )
+})`
+  margin: 24px;
+`
+
+//https://stackoverflow.com/questions/3177836/how-to-format-time-since-xxx-e-g-4-minutes-ago-similar-to-stack-exchange-site
+function timeSince(date: Date): string {
+  var seconds = Math.floor((Date.now() - Number(date)) / 1000)
+
+  var interval = seconds / 31536000
+
+  if (interval > 2) {
+    return Math.floor(interval) + ' years'
+  }
+  interval = seconds / 2592000
+  if (interval > 2) {
+    return Math.floor(interval) + ' months'
+  }
+  interval = seconds / 86400
+  if (interval > 2) {
+    return Math.floor(interval) + ' days'
+  }
+  interval = seconds / 3600
+  if (interval > 2) {
+    return Math.floor(interval) + ' hours'
+  }
+  interval = seconds / 60
+  if (interval > 2) {
+    return Math.floor(interval) + ' minutes'
+  }
+  return Math.floor(seconds) + 2 + ' seconds'
 }
+
+const ReviewCard = styled(({ review, className }: { review: Review; className?: string }) => {
+  return (
+    <div className={className}>
+      <ReviewTopRow>
+        <Avatar src={review.avatarImage}></Avatar>
+        <ShowOnLaptop>
+          <Comment review={review} />
+        </ShowOnLaptop>
+        <RatingList>
+          {Object.entries(review.ratings).map((rating) => {
+            return <Rating rating={rating} key={rating[0]}></Rating>
+          })}
+        </RatingList>
+      </ReviewTopRow>
+      <HideOnLaptop>
+        <Comment review={review} />
+      </HideOnLaptop>
+    </div>
+  )
+})`
+  margin-top: 15px;
+  margin-bottom: 15px;
+  border-radius: 8px;
+  overflow: hidden;
+  width: 100%;
+
+  // A subtle shadow around the card
+  box-shadow: 0px 4px 3px 2px ${colours.neutralLight1};
+`
+
+const RatingLabel = styled.p`
+  margin-bottom: 5px;
+  margin-top: 5px;
+  margin-right: 5px;
+  padding-left: auto;
+  flex: 1;
+  flex-grow: 1;
+  font-weight: 600;
+  width: auto;
+  color: ${colours.neutralDark2};
+`
+
+const Rating = styled(({ className, rating }: { className?: string; rating: [string, number] }) => {
```

Comment:
Can we pass `rating` and `number` in as separate params here? Using array indexing to access them feels weird

Reply:
Done